### DISCUSSION TITLE
Confirming the signing key

### DISCUSSION THREAD

**Discussion by lazka** [openssl/openssl#25371] [Q&A] [answered]
Just to be sure, the signing key on this website is the right one https://openssl-library.org/source/index.html? And source tarballs are no longer available on https://openssl.org/ ?

Since the signing key, the domain for providing the key, and the tarball location changed, and I can't find any information on this change.

**Comment by t8m** [comment, marked as answer]
Yes, [BA54 73A2 B058 7B07 FB27 CF2D 2160 94DF D0CB 81EF](https://keys.openpgp.org/search?q=openssl%40openssl.org) is the signing key.

It was already mentioned in https://github.com/openssl/openssl/blob/openssl-3.3.1/doc/fingerprints.txt in the patch releases in June.

And yes, the only distribution source for openssl releases is now https://github.com/openssl/openssl/releases

**Reply by lazka** [reply]
thanks!
